app/templates/views/guidance/using-notify/trial-mode.html

```diff
@@ -11,41 +11,30 @@
 
   <h1 class="heading-large">Trial mode</h1>
 
-  <p class="govuk-body">When you add a new service it will start in trial mode. This lets you try out GOV.UK Notify for free, with a few restrictions.</p>
-  <p class="govuk-body">While your service is in trial mode you can only:</p>
-  <ul class="govuk-list govuk-list--bullet">
-    <li>send messages to yourself and other people in your team</li>
-    <li>send {{ email_and_sms_daily_limit | message_count('email') }} per day</li>
-    <li>send {{ email_and_sms_daily_limit | message_count('sms') }} per day</li>
-    <li>create letter templates, but not send them</li>
-  </ul>
+  <p class="govuk-body">When you add a new service it will start in trial mode.</p>
+  <p class="govuk-body">While your service is in trial mode, you can only send messages to yourself and your team members.</p>
+  <p class="govuk-body">There’s a daily limit of {{ email_and_sms_daily_limit | message_count('email') }} emails and {{ email_and_sms_daily_limit | message_count('sms') }} text messages.</p>
```

Comment:
The formatter used adds in the word "emails" and "text messages", so the way we had things caused the sentence to read "There’s a daily limit of 50 emails emails... "


```suggestion
  <p class="govuk-body">There’s a daily limit of {{ email_and_sms_daily_limit | message_count('email') }} and {{ email_and_sms_daily_limit | message_count('sms') }}.</p>
```